File: 1812-determine-color-of-a-chessboard-square/1812-determine-color-of-a-chessboard-square.py
class Solution(object):
    def squareIsWhite(self, coordinates):
        """
        :type coordinates: str
        :rtype: bool
        """
        c = coordinates
        if (c == "a2" or c == "a4" or c == "a6" or c == "a8" or
            c == "b1" or c == "b3" or c == "b5" or c == "b7" or
            c == "c2" or c == "c4" or c == "c6" or c == "c8" or
            c == "d1" or c == "d3" or c == "d5" or c == "d7" or
            c == "e2" or c == "e4" or c == "e6" or c == "e8" or
            c == "f1" or c == "f3" or c == "f5" or c == "f7" or
            c == "g2" or c == "g4" or c == "g6" or c == "g8" or
            c == "h1" or c == "h3" or c == "h5" or c == "h7"):
            return True
        return False

File: 1812-determine-color-of-a-chessboard-square/1812-determine-color-of-a-chessboard-square.java
class Solution {
    public boolean squareIsWhite(String c) {
        if(c.equals("a2")||c.equals("a4")||c.equals("a6")||c.equals("a8")||
           c.equals("b1")||c.equals("b3")||c.equals("b5")||c.equals("b7")||
           c.equals("c2")||c.equals("c4")||c.equals("c6")||c.equals("c8")||
           c.equals("d1")||c.equals("d3")||c.equals("d5")||c.equals("d7")||
           c.equals("e2")||c.equals("e4")||c.equals("e6")||c.equals("e8")||
           c.equals("f1")||c.equals("f3")||c.equals("f5")||c.equals("f7")||
           c.equals("g2")||c.equals("g4")||c.equals("g6")||c.equals("g8")||
           c.equals("h1")||c.equals("h3")||c.equals("h5")||c.equals("h7")){
            return true;
           }
           return false;
    }
}
